ZenPacks/zenoss/OpenStackInfrastructure/Endpoint.py

```diff
@@ -164,8 +163,39 @@ def public_keystone_apiendpoint(self):
         except AttributeError:
             return None
 
+    def zenopenstack_url(self, https=False):
+        try:
+            from Products.ZenUtils.controlplane.application import getConnectionSettings
+            from Products.ZenUtils.controlplane import ControlPlaneClient
+        except ImportError:
+            return
```

Comment:
Include a log here or where used for debugging..

Reply:
what isn't logged that should be?